lib/cisco_node_utils/bgp_af.rb

```diff
@@ -59,7 +59,10 @@ def self.afs
     end
 
     def create
-      Feature.bgp_enable if platform == :nexus
+      if platform == :nexus
+        Feature.bgp_enable
+        Feature.nv_overlay_evpn_enable if @safi[/evpn/]
```

Comment:
Yes, because `advertise_l2vpn_evpn` can exist in a unicast safi.

The change above is necessary for creating `address-family l2vpn evpn`.